imports/clipping/Clipping.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import actions from './actions';
+
+class Clipping extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      percentile: [0.9, 0.925, 0.95, 0.96, 0.97, 0.98, 0.99, 0.995, 0.999, 1.0],
+      autoClip: [false, true],
+    };
+  }
+
+  handleClip = (percentile) => {
+    this.props.dispatch(actions.updateClipping(percentile));
+    this.props.dispatch(actions.getClipState());
+  }
+
+  handleAutoClip = (autoClip) => {
+    this.props.dispatch(actions.updateAutoClipping(autoClip));
+    this.props.dispatch(actions.getAutoClipState());
+  }
+
+  render() {
+    const percentiles = this.state.percentile;
+    let percentileButtons = null;
+    percentileButtons = percentiles.map(percentile =>
+      (<button className="button" onClick={() => this.handleClip(percentile)} type="button">
```

Comment:
className="button" -> className="clippingButton" and change the according setting in index.less